transformations/chinese_butter_fingers_perturbation/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "chinese_butter_fingers_perturbation",
+  "test_cases": [
+    {
+      "class": "ChineseButterFingersPerturbation",
+      "inputs": {
+        "sentence": "Andrew finally returned the French book to Chris that I bought last week"
+      },
```

Comment:
I am assuming this is still work in progress?

Reply:
Hi Kaustubh,

Yes, I have an initial version working with a few Chinese characters with a similar approach to the original butter fingers pertubration but that doesn't work well with Chinese characters since there are over 20000 characters compared to 26 alphabets in English.

Therefore, I still working on a version where I can do the perturbation based on a database of Chinese characters. I'll push it as soon as I am done. :)